src/libraries/Microsoft.PowerFx.Interpreter/Values/LambdaFormulaValue.cs

```diff
@@ -15,12 +15,23 @@ internal class LambdaFormulaValue : FormulaValue
     {
         private readonly IntermediateNode _tree;
 
+        internal EvalVisitor Runner { get; }
+
+        internal EvalVisitorContext Context { get; }
+
         // Lambdas don't get a special type. 
         // Type is the type the lambda evaluates too. 
         public LambdaFormulaValue(IRContext irContext, IntermediateNode node)
+            : this(irContext, node, null, null)
+        {
+        }
+
+        public LambdaFormulaValue(IRContext irContext, IntermediateNode node, EvalVisitor visitor, EvalVisitorContext? context)
```

Comment:
>? [](http://example.com/codeflow?start=117&length=1)

non-nullable?
If they don't have this, they'll just call the other ctor overload.